checker/manual/fenum-checker.tex

```diff
@@ -120,9 +118,24 @@ \section{Running the Fenum Checker\label{fenum-running}}
 
 \begin{alltt}
   javac -processor org.checkerframework.checker.fenum.FenumChecker
-        \textit{-Aquals=myproject.qual.MyFenum} MyFile.java ...
+        \textit{-Aquals=qual.MyFenum} MyFile.java ...
 \end{alltt}
 
+This option specifies a comma-no-space-separated list of the fully-qualified
+class names of the annotations.
+
+You can also provide the fully-qualified paths to a set of directories
+that contain the qualifiers using the \code{-AqualDirs} option, and add
+the directories to the boot classpath, for example:
+
+\begin{alltt}
+  javac -Xbootclasspath/p:\textit{/full/path/to/myProject:/full/path/to/otherLib} \ttbs
```

Comment:
The colons shouldn't be in textit font because they are literal text the user should write as is.  (Same comment two lines down.)